carbon-design-system/carbon-addons-iot-react
# fix(valuecard): set of related value card fixes

Closes #2188 

**Summary**

- This is a set of value card fixes related to showing Value Cards in a dashboard, and a few prop types fixes to stop errors from being reported

**Change List (commits, features, bugs, etc)**

- fix(DashboardEditor/DashboardEditorHeader): allow Nodes not just strings in the title
- fix(valueCardRenderer): don't truncate at a max-width if we don't need to
- fix(LayoutConstants): SMALLFULL should go full screen even at small breakpoint size

**Acceptance Test (how to verify the PR)**

- Verify that long string values in the Value card take up the full width available to them
- Verify that if you reduce the font size to 20 or lower, that the long strings still wrap to the next line
